grammars/go.cson

```diff
@@ -162,19 +169,37 @@
         "match": "(\\+|&|\\+=|&=|&&|==|!=|\\-|\\||\\-=|\\|=|\\|\\||<|<=|\\*|\\^|\\*=|\\^=|<\\-|>|>=|/|<<|/=|<<=|\\+\\+|=|:=|;|%|>>|%=|>>=|\\-\\-|!|\\.\\.\\.|&\\^|&\\^=)"
       }
       {
-        "name": "keyword.operator.bracket.go",
-        "match": "(\\(|\\)|\\[|\\]|\\{|\\})"
+        'match': '\\{|\\}'
+        'name': 'meta.brace.curly.go'
       }
       {
-        "name": "keyword.operator.punctuation.go",
-        "match": "(\\.|,|:)"
+        'match': '\\(|\\)'
+        'name': 'meta.brace.round.go'
+      }
+      {
+        'match': '\\[|\\]'
+        'name': 'meta.brace.square.go'
       }
     ]
   'variables':
     'patterns': [
       {
-        'name': 'variable.go'
-        'match': '\\b[\\w_][\\w\\d_]*\\b',
+        'match': '\\b[\\w_][\\w\\d_]*\\b'
+      }
+    ]
+  'delimiters':
```

Comment:
Check... Even through these (and the braces) operators are not documents in more detail by the language specs, they really are still operators. The could (and maybe should) have been documented more along these lines for example: https://msdn.microsoft.com/en-us/library/6a71f45d.aspx

But then again, I agree that as they are *not* described and other language packages seem to describe them as delimiters I think the scopes like `meta.delimiter.comma.go`, `meta.delimiter.period.go`, `meta.brace.curly.go` and `meta.brace.square.go` these should be ok...